societe-archeologique-de-bordeaux.csl

```diff
@@ -22,9 +22,9 @@
         <single>dir.</single>
         <multiple>dir.</multiple>
       </term>
-      <term name="editortranslator" form="short">
-        <single>éd.</single>
-        <multiple>éd.</multiple>
+      <term name="editortranslator">
```

Comment:
I would like it to appear when you have for example the edition of a text in latin with the french traduction (and comments in french).
For example : The invectives of Sallust and Cicero : critical edition with introduction, translation, and commentary by Anna A. Novokhatko